ember-headless-form/package.json

```diff
@@ -21,6 +21,9 @@
     "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
     "lint:js:fix": "eslint . --fix",
     "lint:types": "glint",
+    "lint:prettier": "prettier -c .",
```

Comment:
as an aside, I _really_ like the workflow of having so much namespaced under `lint`. makes the number of commands to remember a lot less, and the `pnpm --filter '*'` at the top makes seeing the output wonderful